<launch>
  <rosparam file="$(find shipbot_ros)/config/realsense_thresholds.yaml" />
  <node name="realsense_node" pkg="shipbot_ros" type="realsense_node" output="screen">
  </node>
</launch>
